pom.xml

```diff
@@ -1877,7 +1877,11 @@
                     <codeserverArgs>
                         <arg>-noprecompile</arg>
                         <arg>-noincremental</arg>
+                        <arg>-generateJsInteropExports</arg>
                     </codeserverArgs>
+                    <compilerArgs>
+                        <arg>-generateJsInteropExports</arg>
```

Comment:
Florent is it really needed to pass `-generateJsInteropExports` option to GWT compiler here to work with Super DevMode?
Or it's enough to pass this option to CodeServer only?

Reply:
@azatsarynnyy when I was using super dev mode without that flag all jsinterop export code was missing so I would say it's needed

Reply:
hmm... Strange behavior. But ok if it works )